groups/groups.yaml

```diff
@@ -742,6 +742,20 @@ groups:
       - thockin@google.com
       - bartek@smykla.com
 
+  - email-id: k8s-infra-staging-scl-image-builder@kubernetes.io
+    name: k8s-infra-staging-scl-image-builder
+    description: |-
+      ACL for Sig Cluster Lifecycle Image Builder project
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
```

Comment:
Email delivery is not used for this group. You can drop this line.

Reply:
Fixed